include/deal.II/hp/fe_collection.h

```diff
@@ -505,6 +505,26 @@ namespace hp
     void
     set_default_hierarchy();
 
+    /**
+     * Returns a sequence of FE indices that corresponds to the registered
+     * hierarchy in ascending order, i.e., FE indices are sorted from lowest to
+     * highest level.
```

Comment:
I had not realized before reading the implementation that the given index can be somewhere in the middle of this sequence. Can you mention this?